util/collate/collate_bench_test.go

```diff
@@ -16,9 +16,8 @@ package collate
 
 import (
 	"math/rand"
-	"testing"
```

Comment:
go-imported doesn't complain on these lines? I don't like unstable format changes over and over though.

Reply:
The change is the other pr. Now I have removed it.

Reply:
BTW，Can you merge https://github.com/pingcap/tidb/pull/31887 ?